lib/src/Dropdown/Dropdown.tsx

```diff
@@ -350,10 +353,10 @@ export const Dropdown: React.FC<DropdownProps> = React.forwardRef(
             }
         }, [props.value, props.placeholder, selectedLabel]);
 
-        React.useEffect(() => {
-            // scroll the focused item into view
-            getOptionsRef()?.[focusedIndex]?.scrollIntoView(false);
-        }, [focusedIndex, getOptionsRef]);
+        // React.useEffect(() => {
+        //     // scroll the focused item into view
+        //     getOptionsRef()?.[focusedIndex]?.scrollIntoView(false);
+        // }, [focusedIndex, getOptionsRef]);
```

Comment:
remove commented code?

Reply:
i believe this code is needed? it helps to scroll the focused item into view

Reply:
omg that's not supposed to be commented out lol I committed the wrong thing let me fix it